src/app/routes/index.test.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { MemoryRouter } from 'react-router-dom';
-import { renderRoutes } from 'react-router-config';
 import { matchPath } from 'react-router';
+import { renderRoutes } from 'react-router-config';
```

Comment:
Note this is a linting side effect - it has changed the order of the imports, but no other changes are required aside from updating `frontPageJson`